### PR TITLE
ttf-indic fonts

### PR DESCRIPTION
**Pull request by akssri**
## Description of changes

Adds ttf-indic-fonts package (derived from [arch](https://gitlab.archlinux.org/archlinux/packaging/packages/ttf-indic-otf)).

## Things done

- Built on platform(s)
  - [x]  x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
I suppose that since you did not fill the maintainers field, you want to to put this package for adoption.
Can I adopt this?

**Comment by akssri** [comment]
Ah, no I'm happy maintaining this. I didn't quite know what the protocol for adding myself for the maintainers list was before listing myself as the maintainer, but it looks like I can do that together with my first contribution (updated the PR accordingly).

File: pkgs/data/fonts/ttf-indic/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.indlinux.org/wiki/index.php/Downloads";
+    description = "Indic Opentype Fonts collection";
+    license = licenses.gpl2Only;
+    maintainers = [ maintainers.akssri ];
+  };
```
Comment (AndersonTorres) [inline pkgs/data/fonts/ttf-indic/default.nix:29]:
```suggestion
  meta = {
    homepage = "https://www.indlinux.org/wiki/index.php/Downloads";
    description = "Indic Opentype Fonts collection";
    license = lib.licenses.gpl2Only;
    maintainers = [ lib.maintainers.akssri ];
    platforms = lib.platforms.all;
  };
```

File: pkgs/data/fonts/ttf-indic/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (AndersonTorres) [inline pkgs/data/fonts/ttf-indic/default.nix:1]: Move this file to `pkgs/by-name/tt/ttf-indic/package.nix`.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Squash and reorder the commits so that you end with this configuration:

1. First commit adding yourself to maintainer-list and message

   `maintainer-list: add akssri`

2. Second  commit with the package file and message

   `ttf-indic: init at 0.2`

File: pkgs/data/fonts/ttf-indic/default.nix (hunk 2 above)
Comment (akssri) [reply pkgs/data/fonts/ttf-indic/default.nix]: done.

**Review by AndersonTorres — APPROVED** [review]

**Review by thiagokokada — APPROVED** [review]

**Comment by AndersonTorres** [comment]
@akssri by the way, welcome to Nixpkgs!

**Comment by akssri** [comment]
Thanks @AndersonTorres  !

**Pull request merged**